src/sbvr-api/uri-parser.ts

```diff
@@ -29,15 +29,19 @@ import * as sbvrUtils from './sbvr-utils';
 export type OdataBinds = ODataBinds;
 
 export interface UnparsedRequest {
+	id?: string;
```

Comment:
I think this needs to become a `requestId` to identify the request from a batch body